app/presenters/tree_builder_ops_rbac.rb

```diff
@@ -18,6 +19,7 @@ def set_locals_for_render
 
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(options)
+    options.merge!(:expand => false)
```

Comment:
kind of strange that we update the incoming options hash here and then not use it.  Unexpected side effect for the caller.